lib/rspec/core/output_wrapper.rb

```diff
@@ -3,17 +3,25 @@ module Core
     # @private
     class OutputWrapper
       # @private
-      attr_writer :output
+      attr_accessor :output
 
       # @private
       def initialize(output)
         @output = output
       end
 
+      def respond_to?(name, priv=false)
+        output.respond_to?(name, priv)
+      end
+
+      def method_missing(name, *args, &block)
+        output.method_missing(name, *args, &block)
+      end
```

Comment:
An issue here is we have behaviour that is tested with a minimal cross section, so it looks like it's working but we don't really have enough test cases for all of our supported behaviour.